proto/serverless/instrumentation/tags/v1/aws.proto

```diff
@@ -0,0 +1,198 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+import "serverless/instrumentation/tags/v1/common.proto";
+
+option go_package = ".;protoc";
+
+message AwsApiGatewayTags {
+  // The AWS Account ID of the API GW.
+  string account_id = 1;
+  // The unique id used for the API GW.
+  string api_id = 2;
+  // The stage of the API GW endpoint that was called.
+  string api_stage = 3;
+
+  message AwsApiGatewayRequestTags {
+    // The unique API GW Request ID.
+    string id = 1;
+    // The request time in milliseconds from epoch.
+    fixed64 time_epoch = 2;
+    // The HTTP protocol of the request.
+    string protocol = 3;
+    // The domain name of the request.
+    string domain = 4;
+    // JSON string containing Request Headers
+    string headers = 5;
+    // The HTTP method of the request.
+    string method = 6;
+    // The HTTP Path of the request.
+    string path = 7;
+
+
+    // JSON string containing Request Path Parameters
+    optional string path_parameters = 16;
+    // JSON string contain Query String Parameters
+    optional string string_parameters = 17;
+  }
+
+  AwsApiGatewayRequestTags request = 15;
+
+}
+
+message AwsLambdaTags {
+  // The architecture of the Lambda function, currently either amd64 or arm64.
+  string arch = 1;
+  // Was the invocation a cold start?
+  bool coldstart = 2;
```

Comment:
I'm torn on this. In general I prefer the question convention, but I don't think we can follow this everywhere since the AWS SDK spans will be one to one with the API responses which do not follow that.

I'm cool to make the change for all our defined spans but not the ones we are defining straight from the library like the AWS SDK.